website/src/components/pages/single-component/_utils.js

```diff
@@ -139,3 +139,27 @@ const IconTitle = ({ icon: Icon, children }) => {
 		</div>
 	);
 };
+
+export const getBrandHeaderColor = (brand, colors) => {
+	let backgroundColor;
+	switch (brand) {
+		case 'WBC':
+			backgroundColor = colors.primary;
+			break;
+		case 'WBG':
+			backgroundColor = colors.primary;
+			break;
+		case 'STG':
+			backgroundColor = colors.hero;
+			break;
+		case 'BSA':
+			backgroundColor = `linear-gradient(to right, ${colors.hero} 0%, #00468e 50%, #00adbd 100%)`;
+			break;
+		case 'BOM':
+			backgroundColor = colors.hero;
+			break;
+		case 'BTFG':
+			backgroundColor = colors.hero;
+	}
+	return backgroundColor;
+};
```

Comment:
An easier way to do this would be:
```js
const styleMap = {
  WBC: COLORS.primary,
  WBG: COLORS.primary,
  BSA: `linear-gradient(to right, ${COLORS.hero} 0%, #00468e 50%, #00adbd 100%)`,
  ...
};
```
and then just call it with your brand:
```sh
css={{
  ...styleMap(BRAND)
}}
```